codecov.yml

```
@@ -3,6 +3,7 @@ coverage:
     project:
       default:
         target: auto
+        threshold: 0.04%
```

You probably also want to add the 100% patch target
```suggestion
    project:
      default:
        target: auto
        threshold: 0.04%
    patch:
      default:
        target: 100%
        threshold: 0%
    
```

---

Another option is changing the PR description from:
> If I understand correctly, this only affects the CodeCov "project" check: the "patch" check will still require that all lines added as part of the PR be covered.

to 
> The "patch" check will still require that the total coverage does not drop due to the added lines

---

updated the description